### ISSUE TITLE
Android guidelines - Support for navigation through components and screen 

### ISSUE THREAD

**Issue by Aksi0m** [infinum/accessibility-mobile-standards#40]
Define and add implementation examples (if applicable) for the given WCAG 2.2 Level A/AA points:

- [x] [1.1.1](https://www.w3.org/WAI/WCAG22/quickref/?currentsidebar=%23col_customize&levels=aaa&showtechniques=111)
- [x] [1.3.1](https://www.w3.org/WAI/WCAG22/quickref/?currentsidebar=%23col_customize&levels=aaa&showtechniques=131)
- [x] [1.3.2](https://www.w3.org/WAI/WCAG22/quickref/?currentsidebar=%23col_customize&levels=aaa&showtechniques=132)
- [x] [1.4.10](https://www.w3.org/WAI/WCAG22/quickref/?currentsidebar=%23col_customize&levels=aaa&showtechniques=1410)
- [x] [2.4.1](https://www.w3.org/WAI/WCAG22/quickref/?currentsidebar=%23col_customize&levels=aaa&showtechniques=241)
- [x] [2.4.3](https://www.w3.org/WAI/WCAG22/quickref/?currentsidebar=%23col_customize&levels=aaa&showtechniques=243)
- [x] [2.4.5](https://www.w3.org/WAI/WCAG22/quickref/?currentsidebar=%23col_customize&levels=aaa&showtechniques=245)
- [x] [2.4.7](https://www.w3.org/WAI/WCAG22/quickref/?currentsidebar=%23col_customize&levels=aaa&showtechniques=247)
- [x] [2.4.11](https://www.w3.org/WAI/WCAG22/quickref/?currentsidebar=%23col_customize&levels=aaa&showtechniques=2411)
- [x] [3.2.1](https://www.w3.org/WAI/WCAG22/quickref/?currentsidebar=%23col_customize&levels=aaa&showtechniques=321)
- [x] [3.2.2](https://www.w3.org/WAI/WCAG22/quickref/?currentsidebar=%23col_customize&levels=aaa&showtechniques=322)
- [x] [3.2.3](https://www.w3.org/WAI/WCAG22/quickref/?currentsidebar=%23col_customize&levels=aaa&showtechniques=323)

Keep in mind that some points are already defined in this repository but it might be that they are still for WCAG 2.1 or they are only written for Android traditional view system. We also need to add compose support where that makes sense.

Your main reference points should be:
1. The [WCAG website](https://www.w3.org/WAI/WCAG22/quickref/?currentsidebar=%23col_customize&levels=aaa) 
2. The iOS implementation of the same point in this repository

**If you find some useful articles, videos, or any resource please share it with the rest of us in our dedicated slack channel.
For any questions you can ask Renato, Petra and Majcen**

**Comment by sttihan** [comment]
This is not closed yet, since it is split into two PRs, thought GitHub would be a bit smarter. 😬
